### PR TITLE
[DevOps] Multiple compliance and quality of life improvements

### PR DESCRIPTION
**Pull request by Tyler-Angell**
### What
Point the GitHub PR build at the ADO PR Build
Use template for actual build process
Added Dependabot for Internal Nugets
Added default added reviewers to SmartPlaces.Facilities codepath
Added CodeQL scanning

### Why
It was pointing at a Buddy build instead.
PR and Main were out of sync
Makes sure Microsoft.SmartPlaces.Facilities nugets stay up to date across projects
Makes sure Microsoft is paying attention to the nugets that are published
Part of SDL requirements

### CODE REVIEW

**Review by joebeernink — APPROVED** [review]

File: .github/workflows/ComplianceScanning.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "ComplianceScanning"
+
+on:
+  push:
```
Comment (hammar) [inline .github/workflows/ComplianceScanning.yml:15]: Should there be a path constraint here as well?

**Review by hammar — APPROVED** [review]
LGTM, with the possible exception of that one comment on paths.

File: .github/workflows/ComplianceScanning.yml (hunk 1 above)
Comment (Tyler-Angell) [reply .github/workflows/ComplianceScanning.yml:15]: Technically the whole repo should have already had this before we even started. So everything needs scanned.

Comment (hammar) [reply .github/workflows/ComplianceScanning.yml:15]: Makes sense!

**Pull request merged**